packages/core/src/routes/connector.ts

```diff
@@ -211,8 +210,11 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
       if (config) {
         validateConfig(config);
       }
-      // Once created, target can not be modified.
-      assertThat(body.metadata?.target === undefined, 'connector.can_not_modify_target');
+
+      if (body.metadata?.target !== undefined) {
```

Comment:
no need for this line. body.metadata?.target === metadata.target

Reply:
The current implementation can let undefined and empty `body.metadata` pass the guard and only block when `body.metadata` has a string-typed value target that does not equal the value of `metadata.target`.

Reply:
Ignore me, the point you mentioned is valid, but some prerequisite is required, will add them in another PR.